packages/material-ui/src/Menu/Menu.d.ts

```diff
@@ -26,9 +26,9 @@ export interface MenuProps extends StandardProps<PopoverProps & Partial<Transiti
    * Override or extend the styles applied to the component.
    */
   classes?: {
-    /** Styles applied to the `Paper` component. */
+    /** Styles applied to the Paper component. */
     paper?: string;
-    /** Styles applied to the `List` component via `MenuList`. */
+    /** Styles applied to the List component.via `MenuList`. */
```

Comment:
```suggestion
    /** Styles applied to the List component via `MenuList`. */
```

Reply:
Good catch. Search and replace SNAFU that I hadn't fixed for that case.